README.md

```diff
@@ -169,6 +169,21 @@ When `dev` is true, the `devDependencies` section is checked as well as the
 `dependencies` section of `package.json`. When `verbose` is true, `jsgl`
 generates more verbose output.
 
+### Use in Gulp
+
+```javascript
+const jsgl = require('js-green-licenses');
+
+gulp.task('check_licenses', function(callback) {
+  const checker = new jsgl.LicenseChecker({
+    dev: true,
+    verbose: false,
+  });
+  checker.defaultHandlers();
+  Promise.resolve(checker.checkLocalDirectory('.'));
```

Comment:
Is `Promise.resolve()` needed? Doesn't this work?
```javascript
gulp.task('check_licenses', function() {
  const checker = new jsgl.LicenseChecker({
    dev: true,
    verbose: false,
  });
  checker.defaultHandlers();
  return checker.checkLocalDirectory('.');
});
```

Reply:
You're correct.  That works.